src/main/java/stsjorbsmod/cards/wanderer/Determination.java

```diff
@@ -37,7 +38,17 @@ public Determination() {
     public void use(AbstractPlayer p, AbstractMonster m) {
         addToBot(new RememberSpecificMemoryAction(p, PrideMemory.STATIC.ID));
         if (!upgraded) {
-            addToBot(new ApplyPowerAction(p, p, new FragilePower(p, magicNumber), magicNumber));
+            if (p.hasPower(FragilePower.POWER_ID)) {
```

Comment:
I suspect we'd need this check to move to an action, rather than happening within the card's `use`, to avoid a "quicksilver + determination" scenario that starts without fragile from adding fragile twice

Reply:
Ah, good catch.